src/components/AssetListItem.vue

```diff
@@ -47,6 +51,10 @@ export default class AssetListItem extends Mixins(TranslationMixin) {
     return formatAddress(this.asset.address, 10);
   }
 
+  handleOpenAssetDetails(asset: Asset): void {
+    this.navigate({ name: RouteNames.WalletAssetDetails, params: { asset } });
```

Comment:
Could you please also `emit` event, for example
`this.$emit('show-details', asset)`

This needed for dex, i want to use in future `AssetList` in our dex views. After catching this event, i will know, that we should to change DEX view to wallet